### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by camsom**
Add basic tests that set up a test wsgi server and postgresql server. ff

@mparent61 @MichaelButkovic f

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - '2.7'
+  - '3.4'
```
Comment (mparent61) [inline .travis.yml]: might as well do 3.5 too... async!!!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+services:
+  - elasticsearch
+install:
+  - travis_retry pip install -r requirements.txt
+  - travis_retry pip install -r requirements-dev.txt
+script:
+  - py.test --cov pageview_digest --cov-report term-missing
+after_success:
+  - coveralls
```
Comment (mparent61) [inline .travis.yml:11]: nice!

**Comment by mparent61** [comment]
LGTM

File: tests/test_application.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            ))
+
+        # Create configured database
+        cursor.execute(
+            "SELECT COUNT(*) FROM pg_database WHERE datname='{}'".format(db_dbname)
+        )
+        if cursor.fetchone()[0] <= 0:
+            cursor.execute("CREATE DATABASE {}".format(db_dbname))
+
+        conn.close()
```
Comment (MichaelButkovic) [inline tests/test_application.py:34]: are lines 34-46 necessary?

Comment (camsom) [inline tests/test_application.py:34]: yes. it is a different database. The initial connection sets up the postgres database and allows me to insert the user/password & database we actually use. 34 - 46 sets up a new connection with the database we actually want the tables from `tables.sql` in

Comment (MichaelButkovic) [inline tests/test_application.py:34]: got it

**Comment by MichaelButkovic** [comment]
:+1:

**Comment by camsom** [comment]
See ya in hell

**Comment by mparent61** [comment]
Just what I need on a Friday morning, Cammy blowing up my inbox.

**Pull request closed without merging**